### ISSUE TITLE
False positive and crashes with Style/RedundantLineContinuation

### ISSUE THREAD

**Issue by eneagoe** [r7kamura/slimcop#16]
It seems that the newly introduced [Style/RedundantLineContinuation](https://github.com/rubocop/rubocop/pull/11694) rule introduces a few false positives and also breaks the auto-correct.

For example on this (partial) slim template

```
.form-group.text-end
  label Currency
  = f.collection_select :currency_code \
    , Money::Currency.all \
    , :iso_code, :description \
    , {include_blank: false} \
    , {class: 'form-control}

```

we get 

```
test.html.slim:4:27: C: [Correctable] Style/RedundantLineContinuation: Redundant line continuation.
    , Money::Currency.all \ ...
                          ^
test.html.slim:5:31: C: [Correctable] Style/RedundantLineContinuation: Redundant line continuation.
    , :iso_code, :description \ ...
                              ^
test.html.slim:6:30: C: [Correctable] Style/RedundantLineContinuation: Redundant line continuation.
    , {include_blank: false} \ ...
                             ^
```

and trying to auto-correct the errors we get
```
slimcop -a test.html.slim
Inspecting 1 file
.rvm/gems/ruby-3.2.2@corpware-webapp/gems/slimi-0.7.4/lib/slimi/parser.rb:571:in `syntax_error!': UnknownLineIndicatorError at /Users/eneagoe/work/external/koen/corpware-webapp/test.html.slim:5:5 (Slimi::Errors::UnknownLineIndicatorError)
    , :iso_code, :description
    ^
```

**Comment by r7kamura** [comment]
hmm, does it look like this cop should be disabled in templates?

**Comment by eneagoe** [comment]
Probably - that's what we did in our project.

**Comment by r7kamura** [comment]
OK, so I'll create a pull request to disable this cop.

By the way, I just noticed that if I try to disable a cop that exists only in the newer RuboCop, I get an error if I am using an earlier version of RuboCop. It seems a bit wasteful to add a restriction on the RuboCop version, but I can't think of any other solution, so I will do so 🤔

**Comment by r7kamura** [comment]
Just fixed at https://github.com/r7kamura/slimcop/pull/17 and released v0.18.0 👌

**Comment by eneagoe** [comment]
Thank you for the quick update!
